tests/testthat/test-empty_audit_file.R

```diff
@@ -0,0 +1,5 @@
+test_that("returns a data frame with 1 row filled with `NA`s", {
+  expect_s3_class(empty_audit_file(), class = "data.frame")
```

Comment:
For the `empty_*` functions, would it be better to use `expect_identical()` so that we can catch the columns as well?

Reply:
no... my expectation/suspicion is that whatever column names are there now are not fixed forever